### PR TITLE
cloud: manage user profiles

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md [hunk 1]
```diff
@@ -46,6 +46,8 @@
 - Manage User Access
   - [Sign In](/tidb-cloud/manage-user-access.md)
   - [Sign Out](/tidb-cloud/manage-user-access.md#sign-out)
+  - [Manage User Passwords](/manage-user-access#manage-user-passwords)
+  - [Manage User Profiles](/manage-user-access#manage-user-profiles)
```
Comment (qiancai) [inline TOC-tidb-cloud.md:50]:
```suggestion
  - [Manage User Passwords](/tidb-cloud/manage-user-access#manage-user-passwords)
  - [Manage User Profiles](/tidb-cloud/manage-user-access#manage-user-profiles)
```

**Review by zhoubasten — APPROVED** [review]
LGTM

**Review by zhoubasten — APPROVED** [review]
LGTM

File: TOC-tidb-cloud.md [hunk 2]
```diff
@@ -46,6 +46,8 @@
 - Manage User Access
   - [Sign In](/tidb-cloud/manage-user-access.md)
   - [Sign Out](/tidb-cloud/manage-user-access.md#sign-out)
+  - [Manage User Passwords](/tidb-cloud/manage-user-access#manage-user-passwords)
+  - [Manage User Profiles](/tidb-cloud/manage-user-access#manage-user-profiles)
```
Comment (TomShawn) [inline TOC-tidb-cloud.md:50]:
```suggestion
  - [Manage User Passwords](/tidb-cloud/manage-user-access.md#manage-user-passwords)
  - [Manage User Profiles](/tidb-cloud/manage-user-access.md#manage-user-profiles)
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 3]
```diff
@@ -36,13 +36,7 @@ To connect to your TiDB cluster via standard connection, perform the following s
...
     3. Click **Create Filter** to confirm the changes.
 
-4. Use a SQL client to connect to TiDB.
-
-    {{< copyable "shell" >}}
-
-    ```shell
-    mysql -u root -h <endpoint> -P <port number> -p
-    ```
+4. Under **Step 2: Connect with a SQL client** in the dialog box, click the tab of your preferred connection method, and then connect to your cluster with the connection string.
```
Comment (TomShawn) [inline tidb-cloud/connect-to-tidb-cluster.md:39]:
```suggestion
4. In the **Connect to TiDB** dialog box displayed in "Step 2: Click **Connect**", click the tab of your preferred connection method, and then connect to your cluster with the connection string.
```

File: tidb-cloud/manage-user-access.md [hunk 4]
```diff
@@ -45,6 +45,16 @@ To change the password, take the following steps:
 
 If your password is not changed within 90 days, you will get a prompt to reset your password when you log in to TiDB Cloud. It is recommended that you follow the prompt to reset the password.
 
+## Manage user profiles
+
+In TiDB Cloud, you can easily mange your profile, including your first name, last time, company name, country, and phone number.
```
Comment (TomShawn) [inline tidb-cloud/manage-user-access.md:50]:
```suggestion
In TiDB Cloud, you can easily manage your profile, including your first name, last time, company name, country, and phone number.
```

File: tidb-cloud/manage-user-access.md [hunk 5]
```diff
@@ -45,6 +45,16 @@ To change the password, take the following steps:
...
+
+1. Click the account name in the upper-right corner of the window.
```
Comment (TomShawn) [inline tidb-cloud/manage-user-access.md:52]: Users might not read this section from Line 1 to Line 52. I am wondering whether it is necessary to clarify what "the window" is...

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 6]
```diff
@@ -92,29 +92,21 @@ You can either create a free [Developer Tier (Dev Tier)](/tidb-cloud/select-clus
 
 2. Click **Connect**. The **Connect to TiDB** dialog box is displayed.
 
-3. In the **Connect to TiDB** dialog box, copy the connection string in **Step 2: Connect with a SQL client** . For example:
+3. Under **Step 2: Connect with a SQL client** in the dialog box, click the tab of your preferred connection method, and then connect to your cluster with the connection string.
```
Comment (TomShawn) [inline tidb-cloud/tidb-cloud-quickstart.md:95]:
```suggestion
3. In the **Connect to TiDB** dialog box displayed in the preceding step "Click **Connect**", click the tab of your preferred connection method, and then connect to your cluster with the connection string.
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

File: tidb-cloud/connect-to-tidb-cluster.md (hunk 3 above)
Comment (qiancai) [reply tidb-cloud/connect-to-tidb-cluster.md:39]: "Step 2: Click **Connect**" is in the the **Connect to TiDB** dialog box.

File: tidb-cloud/tidb-cloud-quickstart.md (hunk 6 above)
Comment (qiancai) [reply tidb-cloud/tidb-cloud-quickstart.md:95]: ditto.

**Review by TomShawn — APPROVED** [review]
LGTM

File: tidb-cloud/manage-user-access.md (hunk 5 above)
Comment (qiancai) [reply tidb-cloud/manage-user-access.md:52]: "the window" means the window of the TiDB Cloud console, this is the only window used in TiDB Cloud docs.

File: tidb-cloud/release-notes-2022.md [hunk 7]
```diff
@@ -8,6 +8,12 @@ aliases: ['/tidbcloud/beta/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) in 2022.
 
+## June 21, 2022
+
+* Support [updating user profiles](/manage-user-access.md#manage-user-profiles) on the TiDB Cloud console, including first name, last time, company name, country, and phone number.
```
Comment (TomShawn) [inline tidb-cloud/release-notes-2022.md:13]:
```suggestion
* Support [updating user profiles](/tidb-cloud/manage-user-access.md#manage-user-profiles) on the TiDB Cloud console, including first name, last time, company name, country, and phone number.
```

File: tidb-cloud/manage-user-access.md (hunk 5 above)
Comment (TomShawn) [reply tidb-cloud/manage-user-access.md:52]: "the upper-right corner of the TiDB Cloud console" or "the upper-right corner of the console" LGTM

**Pull request merged**